modules/utils/2.0/config/default.yaml

```diff
@@ -0,0 +1,34 @@
+## Lines commented out with `#!` are required for the module to run
+## Lines commented out with `#?` can optionally be user-configured
+## Lines commented out with `##` act as regular comments
+
+lcr-modules:
+    utils:
+        paired_modules: ["bwa_mem", "star"]
```

Comment:
What does this do? It's a new field, so you might want to add a comment for the user.